### PR TITLE
added more reference guides

### PR DESCRIPTION
**Pull request by alanngo**
Can you merge into your repo?

### CODE REVIEW

**Comment by alanngo** [comment]
Applied changes

**Comment by alanngo** [comment]
applied changes

File: github/README.md [hunk 1]
```diff
@@ -30,6 +30,25 @@ For my teams, we have one repo and everyone is added on as a [collaborator](http
...
+4. $ cd <PATH TO LOCAL REPO>
+5. $ touch README.md
+6. $ git init
+7. $ git add .
+8. $ git commit - m "initial commit"
+9. $ git remote add origin <REMOTE REPO>
+10. $ git push -u -f origin master
+11. $ git pull
+
+## How to continuously integrate your work w/ remote
```
Comment (kdesai2018) [inline github/README.md:46]:
```suggestion
## How to continuously integrate your work with the remote branch on Github
```

File: github/README.md [hunk 2]
```diff
@@ -30,6 +30,25 @@ For my teams, we have one repo and everyone is added on as a [collaborator](http
...
+6. $ git init
+7. $ git add .
+8. $ git commit - m "initial commit"
+9. $ git remote add origin <REMOTE REPO>
+10. $ git push -u -f origin master
+11. $ git pull
+
+## How to continuously integrate your work with the remote branch on Github
+1. $ git add <file name>
+2. $ git commit -m "<message>"
```
Comment (kdesai2018) [inline github/README.md:48]:
```suggestion
2. $ git commit -m "detailed commit message here"
```

**Pull request merged**